proof_of_stake/src/epoched.rs

```diff
@@ -544,6 +584,27 @@ where
         }
     }
 
```

Comment:
The function `rev_update_while`, which is replaced by the `rev_while` added here, does not seem to be used anymore. Should it be removed?

Reply:
good point! yeah, let's remove it